cachematrix.R

```diff
@@ -1,15 +1,35 @@
-## Put comments here that give an overall description of what your
-## functions do
+## These functions generate, cache, and return the cache of the inverse
```

Comment:
Generates... from what? It is worth calling out what the expected inputs are and what the default behavior is if inputs are not provided.